### PR TITLE
Made navbar buttons unselectable

### PR DESCRIPTION
**Pull request by polybuildr**
There is no reason for the buttons to be selectable, added a CSS
property that disallows selecting.

### CODE REVIEW

File: css/teaser.css [hunk 1]
```diff
@@ -85,6 +85,7 @@ body {
   height: 100px;
   margin: 0 auto;
   position: relative;
+  -webkit-user-select: none;
```
Comment (nisargjhaveri) [inline css/teaser.css:88]: Shouldn't we add prefixes for other browsers too? This won't work on Firefox or other major browsers.

**Comment by polybuildr** [comment]
Added other vendor prefixes too, along with the unprefixed version.

**Comment by nisargjhaveri** [comment]
Still doesn't work for Opera. But, I think it is fine. :P

Thanks.

**Comment by polybuildr** [comment]
Opera supports the `-webkit-` prefix from the last 6 versions or so. I think that's enough support. :P

**Comment by nisargjhaveri** [comment]
Oh. Sorry, my bad.

**Pull request merged**